### PR TITLE
fix isPackagePrivateClass (and test for same)

### PR DESCRIPTION
**Pull request by dturner-tw**

### CODE REVIEW

**Comment by dturner-tw** [comment]
Fixes https://github.com/pantsbuild/pants/issues/942

File: test/com/sun/tools/jmake/PCDEntryTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                            1000000000000L,
+                            null);
+
+    }
+
+    @Test
+    public void testIsPackagePrivateClass() {
+        assertEquals(false, createPCDEntry("foo/bar/Foo", "/tmp/foo/bar/Foo.java").isPackagePrivateClass());
+        assertEquals(false, createPCDEntry("foo/bar/Foo$1", "/tmp/foo/bar/Foo.java").isPackagePrivateClass());
+        assertEquals(true, createPCDEntry("foo/bar/Bar", "/tmp/foo/bar/Foo.java").isPackagePrivateClass());
```
Comment (ericzundel) [inline test/com/sun/tools/jmake/PCDEntryTest.java]: given this example, I'm not sure what a package private class really is supposed to be.

Seems like in this case:

Foo.java
````
public class Foo {
}

class Bar {
   class Baz {
   }
}
```

that Bar would be considered package private, but Bar.Baz wouldn't be?

Comment (dturner-tw) [inline test/com/sun/tools/jmake/PCDEntryTest.java]: I'm not sure why Bar.Baz (Bar$1) wouldn't be.  I'm at the wrong computer
right now, so I can't actually run the test, but I think that's what it
would show, as well.

Comment (ericzundel) [inline test/com/sun/tools/jmake/PCDEntryTest.java]: nm, I think you are right

**Comment by dturner-tw** [comment]
Moved to RBcommons and submitted.

**Pull request closed without merging**